packages/orbit-components/src/Collapse/Collapse.stories.tsx

```diff
@@ -238,12 +244,10 @@ export const Rtl: Story = {
               Clear
             </TextLink>
           }
+          expandButtonLabel="Expand"
+          collapseButtonLabel="Collapse"
         >
-          <ChoiceGroup
-            filter
```

Comment:
Why was this deleted?

Reply:
The `filter` prop was doing nothing. For the purpose of the story, it was useless and weird when one hovered the last part of it
<img width="186" alt="image" src="https://github.com/user-attachments/assets/53eb7a57-0998-4f98-bf69-ed27baa97254" />